metricsaggregator/src/main/java/io/dockstore/metricsaggregator/client/cli/Client.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright 2023 OICR and UCSC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *           http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.dockstore.metricsaggregator.client.cli;
+
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.MissingCommandException;
+import com.beust.jcommander.ParameterException;
+import io.dockstore.metricsaggregator.MetricsAggregatorConfig;
+import io.dockstore.metricsaggregator.MetricsAggregatorS3Client;
+import io.dockstore.metricsaggregator.client.cli.CommandLineArgs.AggregateMetricsCommand;
+import io.dockstore.openapi.client.ApiClient;
+import io.dockstore.openapi.client.Configuration;
+import io.dockstore.openapi.client.api.ExtendedGa4GhApi;
+import java.io.File;
+import java.net.URISyntaxException;
+import java.util.Optional;
+import org.apache.commons.configuration2.INIConfiguration;
+import org.apache.commons.configuration2.builder.fluent.Configurations;
+import org.apache.commons.configuration2.ex.ConfigurationException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Client {
```

Comment:
`Client` a little generic, used in too many places. 
Don't know if we'll import this anywhere, but just in case should probably make more unique name

Reply:
Changed to `MetricsAggregatorClient`